src/js/angular/graphexplore/controllers/rdf-class-hierarchy.controller.js

```diff
@@ -23,6 +23,9 @@ angular
 RdfClassHierarchyCtlr.$inject = ["$scope", "$rootScope", "$location", "$repositories", "$licenseService", "$window", "toastr", "GraphDataRestService", "UiScrollService", "RdfsLabelCommentService", "$timeout", "ModalService", "bowser", "LocalStorageAdapter", "LSKeys", "RDF4JRepositoriesRestService", "$translate"];
 
 function RdfClassHierarchyCtlr($scope, $rootScope, $location, $repositories, $licenseService, $window, toastr, GraphDataRestService, UiScrollService, RdfsLabelCommentService, $timeout, ModalService, bowser, LocalStorageAdapter, LSKeys, RDF4JRepositoriesRestService, $translate) {
+
+    const MAX_LOADED_GRAPHS = 1000;
```

Comment:
Duplication again. Isn't it better if we have a file with exported constants in the models folder for example and reuse them where necessarry?

Reply:
The duplication is because the constants are defined in two separate controllers, and I am not sure they should have the same value for both views. For example, in my environment, the "Class Hierarchy" view is heavier than "Class Relations." When the data is larger, the "Class Hierarchy" view gets blocked, whereas "Class Relations" is slow but fully loaded. We may need to decrease the value for one of the views.

Reply:
Yeah, you are right. I initially didn't noticed that these are in different controllers and later I just forgot to alter my comment.

Reply:
added documentation